packages/babel-plugin/__tests__/transform-stylex-position-try-test.js

```diff
@@ -0,0 +1,297 @@
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+jest.autoMockOff();
+
+const stylexPlugin = require('../src/index');
+const { transformSync } = require('@babel/core');
+
+function transform(source, opts = {}) {
+  const { code, metadata } = transformSync(source, {
+    filename: opts.filename,
+    parserOpts: {
+      flow: 'all',
+    },
+    plugins: [[stylexPlugin, { ...opts }]],
+  });
+
+  return { code, metadata };
+}
+
+describe('@stylexjs/babel-plugin', () => {
+  describe('[transform] CSS position-try', () => {
+    test('position-try object', () => {
+      const { code, metadata } = transform(`
+        import * as stylex from '@stylexjs/stylex';
+        export const name = stylex.positionTry({
+          positionAnchor: '--anchor',
+          top: '0',
+          left: '0',
+          width: '100px',
+          height: '100px'
+        });
+      `);
+
+      expect(code).toMatchInlineSnapshot(`
+        "import * as stylex from '@stylexjs/stylex';
+        export const name = "--x1a7kwev";"
+      `);
+
+      expect(metadata).toMatchInlineSnapshot(`
+        {
+          "stylex": [
+            [
+              "--x1a7kwev",
+              {
+                "ltr": "@position-try --x1a7kwev {position-anchor:--anchor;top:0;left:0;width:100px;height:100px;}",
+                "rtl": null,
+              },
+              1,
+            ],
+          ],
+        }
+      `);
+    });
+
+    test('local variables used in position-try object', () => {
+      const { code, metadata } = transform(`
+        import * as stylex from '@stylexjs/stylex';
+        const SIZE = '100px';
+        export const name = stylex.positionTry({
+          positionAnchor: '--anchor',
+          top: '0',
+          left: '0',
+          width: SIZE,
+          height: SIZE
+        });
+      `);
+
+      expect(code).toMatchInlineSnapshot(`
+        "import * as stylex from '@stylexjs/stylex';
+        const SIZE = '100px';
+        export const name = "--x1a7kwev";"
+      `);
+
+      expect(metadata).toMatchInlineSnapshot(`
+        {
+          "stylex": [
+            [
+              "--x1a7kwev",
+              {
+                "ltr": "@position-try --x1a7kwev {position-anchor:--anchor;top:0;left:0;width:100px;height:100px;}",
+                "rtl": null,
+              },
+              1,
+            ],
+          ],
+        }
+      `);
+    });
+
+    test('template literals used in position-try object', () => {
+      const { code, metadata } = transform(`
+        import * as stylex from '@stylexjs/stylex';
+        const SIZE = '100px';
+        const name = stylex.positionTry({
+          positionAnchor: '--anchor',
+          top: '0',
+          left: '0',
+          width: SIZE,
+          height: SIZE
+        });
+        export const styles = stylex.create({
+          root: {
+            positionTryFallbacks: \`\${name}\`,
+          }
+        });
+      `);
```

Comment:
This fixture isn't really testing "template literals used in position-try object". It's testing literals in `create`, which we already test elsewhere. I noticed there's also a keyframes test like this, so probably should update that one too.